packages/appkit/src/utils/HelpersUtil.ts

```diff
@@ -222,6 +222,30 @@ export const WcHelpersUtil = {
       typeof data.params.event === 'object' &&
       data.params.event !== null
     )
+  },
+
+  isOriginAllowed(
```

Comment:
Do we have unit tests for this function, might be nice to test with different inputs